view/adminhtml/templates/element/navigation.phtml

```diff
@@ -0,0 +1,107 @@
+<?php
+/** @var \Magento\Framework\Escaper $escaper */
+/** @var \Hyva\Admin\Block\Grid\Navigation $block */
```

Comment:
Please adjust the type hint to be a regular `\Magento\Framework\View\Element\Template`.